README.md

```diff
@@ -30,10 +30,13 @@ mkdir -p $HOME/.eigenlayer/eigenda/db
 ### Operator Networking Security Setup
 Retrieval Setup:
 
-In order for users to retrieve data from your node, you will need to open access to retrieval ports.
+In order for users to retrieve data from your node, you will need to open access to retrieval ports. 
 
 Ensure the port specified as `NODE_RETRIEVAL_PORT` in the [.env](https://github.com/Layr-Labs/eigenda-operator-setup/blob/master/.env#L17) has open access to the public internet.
 
+For users with private IP e.g. connecting to the Internet via a router, you may need to perform port forwarding to open the retrieval ports. 
```

Comment:
This probably should be true for Dispersal ports too right? Maybe move it after disperal to make it a common instruction.

Reply:
Done!